cypress/integration/dashboard.spec.ts

```diff
@@ -1,54 +1,66 @@
-describe('DashboardUserChoice Test', () => {
-  it('Should have 5 Choices to navigate My Goals/My Feelings/My Time/Explore/ZinZen', () => {
-    cy.visit('/');
-    cy.get('.lang-btn1').contains('English').click();
-    cy.get('.theme-choice-btn').first().click();
-    cy.contains('I already know ZinZen').click();
-    cy.get('.dashboard-choice-light1').should('have.length', 1);
-    cy.get('.dashboard-choice-light').should('have.length', 4);
+describe("DashboardUserChoice Test", () => {
+  it("Should have 5 Choices to navigate My Goals/My Feelings/My Time/Explore/ZinZen", () => {
+    cy.visit("/");
+    cy.get(".lang-btn1").contains("English").click();
+    cy.get(".theme-choice-btn").first().click();
+    cy.contains("I already know ZinZen").click();
+    cy.get(".dashboard-choice-light1").should("have.length", 1);
+    cy.get(".dashboard-choice-light").should("have.length", 4);
   });
 
-  it('Should have option to add Goals and Feelings', () => {
-    cy.get('.add-icon').should('have.length', 2);
+  it("Should have option to add Goals and Feelings", () => {
+    cy.get(".add-icon").should("have.length", 2);
     // cy.get('.add-icon').eq(0).click().location('pathname')
     // .should('include', 'MyGoals')
     // .go('back');
-    cy.get('.add-icon').eq(1).click().location('pathname')
-      .should('include', 'AddFeelings')
-      .go('back');
+    cy.get(".add-icon").eq(1).click().location("pathname").should("include", "AddFeelings").go("back");
```

Comment:
Expected line break before .should. ( eslint )
Did husky skipped or missed it ?

Reply:
Husky is now setup for prettier and not eslint anymore.